src/components/core/pivotTable/agGridData.ts

```diff
@@ -94,6 +94,7 @@ export const getRow = (
         if (isSubtotal && !row.subtotalStyle) {
             row.subtotalStyle = subtotalStyles[rowHeaderIndex];
         }
+        row.isSubtotal = isSubtotal;
```

Comment:
Can we use the same approach as for grand totals? I mean the `type` field.

Reply:
Yes, I've implemented the solution.